src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -47,6 +47,7 @@ class QuerySpansRequestBody(V1RoutesBaseModel):
     end_time: Optional[datetime] = None
     limit: int = DEFAULT_SPAN_LIMIT
     root_spans_only: Optional[bool] = None
+    orphan_span_as_root_span: bool = True
```

Comment:
Should we be defaulting to the faster query path?

While I understand that defaulting to the slower query path impacts first touch. It's not entirely the worst breaking change to me because you still can do what you need to by setting a flag.

I get it though. just a tough tradeoff - would prefer to have a user opt into the slower query path and default to the fast

Maybe a warning message about this?? I want people to move to the faster path.